twiggy/twiggy.rs

```diff
@@ -19,7 +19,7 @@ fn main() {
     let options = opt::Options::from_args();
     if let Err(e) = run(&options) {
         eprintln!("error: {}", e);
-        for c in e.causes().skip(1) {
+        for c in Fail::iter_causes(&e).skip(1) {
```

Comment:
Two things:

1. The `iter_causes` method already skips the first error in the chain, so we can remove the `skip(1)` bit now (yay!)
2. Rather than use unified function calling syntax (UFCS) we should use method calling syntax: `e.iter_causes()`

Reply:
Hi @fitzgen :) Regarding **1:** Oops! Forgot to re-push after reading that `iter_causes()` _"...does the skip(1) dance for us..."_ ;)